app/controllers/items_controller.rb

```diff
@@ -50,16 +50,15 @@ def item_params
   def set_show_headline(item, review_count)
     headline = "Reviews"
     if review_count > 0
-      rating_sum = item.reviews.pluck(:rating).inject { |sum, n| sum + n }
-      rating_count = item.reviews.count
-      average_review = (rating_sum.to_f / rating_count).round(2)
-
-      headline += " - Average Review Score: #{average_review}"
+      headline += " - Average Review Score: #{calculate_average_review(item)}"
     end
     headline
   end
 
-  def calculate_average_review
+  def calculate_average_review(item)
+    rating_sum = item.reviews.pluck(:rating).inject { |sum, n| sum + n }
+    rating_count = item.reviews.count
+    (rating_sum.to_f / rating_count).round(2)
   end
```

Comment:
Notice that you're doing `item.reviews` twice, as well as doing item & review logic in the controller? `average review score` should be an instance method on `Item`. Separation of concerns - put model-related logic in the model.